src/components/Select/AutosuggestMulti/__tests__/AutosuggestMulti.spec.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { AutosuggestMulti } from '../AutosuggestMulti';
+import { SUGGESTIONS, SUGGESTION_TO_STRING } from '../../../Autosuggest/__mocks__/suggestions';
+
+describe('AutosuggestMulti', () => {
+    const suggestionToString = SUGGESTION_TO_STRING;
+    const inputPlaceholder = 'type here...';
+    const mockOnSelectionChange = jest.fn();
+    const mockOnInputValueChange = jest.fn();
+    const mockOnBlur = jest.fn();
+
+    let suggestionsList = [];
+    let selectedSuggestions = [];
+
+    let wrapper;
+    const setFocusOnAutosuggest = () =>
+        wrapper.find('[role="searchbox"]').simulate('keyDown', { key: 'Enter' });
+
+    beforeEach(() => {
+        wrapper = mount(
+            <AutosuggestMulti
+                selectedSuggestions={selectedSuggestions}
+                suggestions={suggestionsList}
+                suggestionToString={suggestionToString}
+                inputPlaceholder={inputPlaceholder}
+                onSelectionChange={mockOnSelectionChange}
+                onInputValueChange={mockOnInputValueChange}
+                onBlur={mockOnBlur}
+            />
+        );
+    });
+
+    afterEach(() => {
+        suggestionsList = [];
+        selectedSuggestions = [];
+        wrapper.unmount();
+    });
+
+    describe('rendering', () => {
+        it('should initially render empty component correctly', () => {
+            expect(toJson(wrapper)).toMatchSnapshot();
+        });
+        it('should render empty component correctly when focused', async () => {
+            setFocusOnAutosuggest();
+            expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
these test can easily be wrong / snapshot changes. It would make sense to add an assertion that actually checks if the component is correctly focused. something like:
```
const inputNode = wrapper.find('input').getDOMNode();
expect(document.activeElement).not.toBe(inputNode);
```

Reply:
Yes, it true. Updated